inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -17,9 +17,9 @@
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
```

Comment:
Nope it will disappear. @nikpap you can remove the brief parts. 

Depending on how complex the logic is, we can either implement it for the brief format here (https://github.com/inspirehep/inspirehep-search-js/blob/master/src/inspirehep-search-js/filters/doi.js) or we do a jinja filter that renders the DOI field and we return the pre-formatted value in the serializer (https://github.com/inspirehep/inspire-next/blob/invenio3/inspirehep/modules/records/serializers/json.py#L49)